rootfs/rootfs/etc/vmware-tools/scripts/vmware/network

```diff
@@ -0,0 +1,596 @@
+#!/bin/sh
```

Comment:
Let me take a deeper look into this, I'm not even 100% sure this file is needed for boot2docker at all.